prepare_setup.sh

```diff
@@ -7,6 +7,8 @@ go_dir=/usr/local
 helm_version="v3.4.0"
 helm_package_dir_name='helm'
 
+kind_version="v0.10.0"
```

Comment:
IMO, pinning it to a certain version would be best to not be surprised of different behavior when preparing new nodes, though we will need to figure out a way to install latest stable Kubenetes in the node image.

Reply:
> IMO, pinning it to a certain version would be best to not be surprised of different behavior when preparing new nodes, though we will need to figure out a way to install latest stable Kubenetes in the node image.

In k8s ci we are using the latest official release to find if there will be any surprises later and I belive it should be the same for kind

Reply:
That is true for deployment components, but kind is a tool we use to deploy those components, so i think pinning kind and building the node image with latest stable Kubernetes is the correct way of doing it. since also latest kind does not necessarily deploy the latest Kubernetes.